example/webdriverio-test/package.json

```diff
@@ -18,7 +18,8 @@
     "appium-xcuitest-driver": "^5.11.0",
     "ts-node": "^10.9.1",
     "typescript": "^5.3.2",
-    "webdriverio": "^8.24.2"
+    "webdriverio": "^8.24.2",
+    "@fishjam-dev/js-server-sdk": "^0.1.0"
```

Comment:
When you modify this file manually, please make sure you are keeping alphabetical order (as `yarn` does).